Week2/Solution/solution_week3.txt

```diff
@@ -0,0 +1,136 @@
+Write a python script to perform CRUD operation on following table of "ESM.db" database.
+
+
+Employee ( eid, ename, dept, basic, branch). eid must be primary key.
+
+Department : Account, Inventory, IT, HR.
+
+Peform Following Operation.
+
+1. Create table
+-->
+       	import sqlite3
+	       con=sqlite3.connect("EMS.db")
+       	q="create table employee(eid int primary key,ename text,dept text,basic int ,branch text)"
+       	cur=con.cursor()
+	       cur.execute(q)

```

Comment:
its indentation error...!!! is it?

Reply:
May be....
Because I have uploaded text file